downstairs/src/region.rs

```diff
@@ -605,11 +649,46 @@ impl Region {
             }
         };
         save_stream_to_file(extent_copy, repair_stream.into_inner()).await?;
+        let mut count = 1;
+
+        // If we are replacing our region with one from an older version
+        // that contained the SQLite, then look for and copy the database
+        // files.
+        for opt_file in &[ExtentType::Db, ExtentType::DbShm, ExtentType::DbWal]
+        {
+            let filename = extent_file_name(eid as u32, opt_file.clone());
+
+            if repair_files.contains(&filename) {
+                let extent_shm = Self::create_copy_file(
+                    copy_dir.clone(),
+                    eid,
+                    Some(opt_file.clone()),
+                )?;
+                let repair_stream = match repair_server
+                    .get_extent_file(eid as u32, opt_file.to_file_type())
+                    .await
+                {
+                    Ok(rs) => rs,
+                    Err(e) => {
+                        crucible_bail!(
+                            RepairRequestError,
+                            "Failed to get extent {} {} file: {:?}",
+                            eid,
+                            opt_file,
+                            e,
+                        );
+                    }
+                };
+                save_stream_to_file(extent_shm, repair_stream.into_inner())
+                    .await?;
+                count += 1;
+            }
+        }
 
         // After we have all files: move the repair dir.
         info!(
             self.log,
-            "Repair files downloaded, move directory {:?} to {:?}",
+            "{count} Repair files downloaded, move directory {:?} to {:?}",
```

Comment:
```suggestion
            "{count} repair files downloaded, move directory {:?} to {:?}",
```

Reply:
Done in [7bef555](https://github.com/oxidecomputer/crucible/commit/7bef55560bd1a345855ca77b6c296616ee66e0fe)